src/labels_by_user_input.py

```diff
@@ -32,3 +37,15 @@ def labels_by_user_input(data, pull):
     if f"/un{LABEL_APPROVE}".lower() in body:
         remove_label(pull=pull, label=LABEL_APPROVE)
         set_commit_status_pending_no_approve(commit=last_commit)
+
+    semver_labels_okay = False
```

Comment:
```
semver_labels_okay = False
    new_labels, remove_labels = check_body_for_semver_labels(body=body)
    if new_labels or remove_labels:
        semver_labels_okay = add_remove_labels(
            pull=pull, new_labels=new_labels, remove_labels=remove_labels
        )
```

this code as well please, the goal if to have here only actions
if something....:
    .....
else: 
    ......

Reply:
only line that must remain prior to the `if` is `check_body_for_semver_labels`, removed additional boolean var.